### PR TITLE
[ADD] sawb_custom_contact_interface

### PR DESCRIPTION
**Pull request by victor-champonnois**
## Description
Custom module for sawb's contact interface

I don't understand the precommit error.

Tests fail because the module depends [on partner_industry_secondary](https://github.com/OCA/partner-contact/pull/1418) wich is not merged yet.

## Odoo task (if applicable)
https://gestion.coopiteasy.be/web#id=9866&action=475&active_id=420&model=project.task&view_type=form&menu_id=536

## Checklist before approval

- [ ] Tests are present (or not needed).
- [ ] Credits/copyright have been changed correctly.
- [ ] Change log snippet is present.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -1,47 +1,2 @@
...
-git+https://github.com/coopiteasy/cie-resource#subdirectory=setup/resource_planning
-git+https://github.com/coopiteasy/cie-resource#subdirectory=setup/resource_activity_delivery
-git+https://github.com/coopiteasy/cie-resource#subdirectory=setup/resource_activity_guide
-git+https://github.com/coopiteasy/account-invoicing@12.0-account_invoice_date_required#subdirectory=setup/account_invoice_date_required
-git+https://github.com/coopiteasy/account-invoicing@12.0-account_invoice_negative_total#subdirectory=setup/account_invoice_negative_total
-git+https://github.com/coopiteasy/cie-timesheet@12.0#subdirectory=setup/hr_timesheet_task_change_project
-git+https://github.com/coopiteasy/cie-timesheet@12.0#subdirectory=setup/hr_timesheet_overtime
-git+https://github.com/coopiteasy/addons@12.0#subdirectory=setup/resource_work_time_from_contracts
-git+https://github.com/coopiteasy/addons@12.0#subdirectory=setup/company_today
-git+https://github.com/fcayre/odoo-usability@12.0-fix-account_usability-and-make-pip-installable#subdirectory=setup/sale_down_payment
```
Comment (remytms) [inline test-requirements.txt:47]: Is that normal that you removed all the test dependencies of this repository ?

**Comment by remytms** [comment]
@victor-champonnois Can you squash your fixup commits. :)

File: test-requirements.txt (hunk 1 above)
Comment (remytms) [reply test-requirements.txt:47]: How, yes it comes from the 12.0 branch, here we are on 16.0.

**Review by remytms — APPROVED** [review]

**Comment by victor-champonnois** [comment]
@remytms thanks for the review !

**Pull request merged**